src/everest/bin/monitor_script.py

```diff
@@ -23,7 +24,7 @@ def monitor_entry(args=None):
     options = parser.parse_args(args)
 
     if options.debug:
-        logging.getLogger().setLevel(logging.DEBUG)
+        logging.getLogger(EVEREST).setLevel(logging.DEBUG)
```

Comment:
These changes are fine, but can you put them in another commit as they dont seem related to the main change?